server/util/timeutil/timeutil.go

```diff
@@ -72,3 +73,13 @@ func StopAndDrainTimer(t *time.Timer) {
 		}
 	}
 }
+
+// TODO: combine StopAndDrainClockworkTimer and StopAndDrainTimer.
```

Comment:
Does changing the argument type in StopAndDrainTimer to accept clockwork.Timer interface not work?